next.config.js

```diff
@@ -18,5 +18,9 @@ module.exports = {
     IUNISWAP_V2_FACTORY_ADDR: '0x5C69bEe701ef814a2B6a3EDD4B1652CB9cc5aA6f',
     IUNISWAP_V2_PAIR_ADDR: '0x1c5DEe94a34D795f9EEeF830B68B80e44868d316',
     IUNISWAP_V2_ROUTER_02_ADDR: '0x7a250d5630B4cF539739dF2C5dAcb4c659F2488D',
+    POSITION_MANAGER_ADDRESS:'0x3eFadc5E507bbdA54dDb4C290cc3058DA8',
```

Comment:
Can you make this so that we can control the variables using "yarn 'env'"?

Reply:
Ok I will check for different environments